tensorflow/lite/micro/kernels/cmsis-nn/conv.cc

```diff
@@ -159,6 +157,19 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   output_dims.w = output->dims->data[2];
   output_dims.c = output_shape.Dims(3);
 
+  // Dynamically allocate per-channel quantization parameters.
+  // TODO(#43475): This allocation is done even for non-int8 cases to get around
+  // a bug in kernel_utils.cc which incorrectly uses per_channel_output_shift in
```

Comment:
the referenced issue doesn't really talk about the bug in kernel_utils.cc

If you could make a separate issue for just that, we can figure out if if can be fixed quickly. And referencing this PR from the newly created issue would also be nice so that we can remember to add the appropriate if check here.

Reply:
Ah.. wrote the wrong ticket number here. It is https://github.com/tensorflow/tensorflow/issues/42883 . I'll fix that. I've made a reference in the issue as well about this PR.

Reply:
The if check and the subsequent memory save would be applicable for the reference implementations of conv and DW conv as well.